### ISSUE TITLE
PHP multiple variable assignment reports error

### ISSUE THREAD

**Issue by roborourke** [humanmade/coding-standards#69]
The rule "Assignments must be first block of code on a line" reports incorrectly when using the following syntax:

```php
$foo = $bar = 'thing';
```

**Comment by rmccue** [comment]
This may be intentional; not sure where that rule is coming from specifically though, what's the error code (run with `-s`)?

**Comment by roborourke** [comment]
It’s not a construct I use in code often if at all, there just happened to be a valid use case at the time. It’s easily avoidable so the solution here could just be to update our engineering handbook to say don’t do this if we think it’s less readable.

**Comment by rmccue** [comment]
I think the code's fine from a readability perspective. I can confirm this is still occurring:

```sh
echo -e "<?php\n\$foo = \$bar = 'hi';" | vendor/bin/phpcs -s --standard=HM -
FILE: STDIN
----------------------------------------------------------------------
FOUND 1 ERROR AFFECTING 1 LINE
----------------------------------------------------------------------
 2 | ERROR | Assignments must be the first block of code on a
   |       | line (Squiz.PHP.DisallowMultipleAssignments.Found)
---------------------------------------------------------------------
```

**Comment by mikeselander** [comment]
OK well we could turn that error off if you all fee. It's actually been handily split off into two bits, which I think is good as we probably don't want to allow assignment in control structures which this rule also disallows.

I'd recommend turning off `Found` which would cover the above case, and keep `FoundInControlStructure`.

The result would look like:

Pass:
`$foo = $bar = 'thing';`

Fail:
`if ( $foo = get_me_something() )`

Thoughts?

**Comment by kadamwhite** [comment]
I like Mike's solution
